dockers/piped-base/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN \
         bash && \
     update-ca-certificates && \
     mkdir ${PIPED_TOOLS_DIR} && \
-    chmod 774 /etc/ssh/ssh_config && \
+    chmod 777 /etc/ssh/ssh_config && \
```

Comment:
do we need the execute permission for this config file 👀

Reply:
For sure, `776` can be enough.